libraries/AP_HAL_SITL/SITL_State.cpp

```diff
@@ -75,13 +75,13 @@ void SITL_State::_sitl_setup(const char *home_str)
 #endif
     fprintf(stdout, "Starting SITL input\n");
 
+    _sitl = SITL::SITL::instance();
     // find the barometer object if it exists
-    _sitl = (SITL::SITL *)AP_Param::find_object("SIM_");
-    _barometer = (AP_Baro *)AP_Param::find_object("GND_");
-    _ins = (AP_InertialSensor *)AP_Param::find_object("INS_");
-    _compass = (Compass *)AP_Param::find_object("COMPASS_");
+    _barometer = AP_Baro::instance();
+    _ins = AP_InertialSensor::get_instance();
```

Comment:
The 4.9.3 compiler was giving errors on a namespace collision with `instance` where as 5.4 was not. I've changed them to match `get_instance` like `AP_InertialSensor`.